test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-1.xml

```diff
@@ -1426,6 +1426,45 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<output type="success" caseSensitive="yes" regex="no" platforms=".*aarch64.*">Current statistics for cache</output>
 	</test>
 
+
```

Comment:
Please remove this duplicate blank line.

Reply:
Fixed.